src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -70,6 +75,12 @@ TEST_CASE("generate load with unique accounts", "[loadgen]")
     auto nodes = simulation->getNodes();
     auto& app = *nodes[0]; // pick a node to generate load
 
+    std::string fileName = "stellar-load-transactions.xdr";
+    auto cleanup = gsl::finally([&]() { std::remove(fileName.c_str()); });
+
+    generateTransactions(app, fileName, nTxs, nAccounts,
+                         /* offset */ nAccounts);
```

Comment:
Nit: use `LOADGEN_PREGENERATED_TRANSACTIONS_FILE` here instead of hard code, you can just copy the path by value in the cleanup lambda.